Core/DefInjected/ThingDef/Buildings_Misc.xml

```diff
@@ -2,105 +2,105 @@
   
   <!-- EN: party spot -->
-  <PartySpot.label>TODO</PartySpot.label>
+  <PartySpot.label>feestplek</PartySpot.label>
   <!-- EN: Designates a spot for throwing parties. -->
-  <PartySpot.description>TODO</PartySpot.description>
+  <PartySpot.description>Wijst een plek aan waar feestjes kunnen worden gegeven.</PartySpot.description>
   
   <!-- EN: pod launcher -->
-  <PodLauncher.label>TODO</PodLauncher.label>
+  <PodLauncher.label>capsule lanceerbasis</PodLauncher.label>
   <!-- EN: A fueling port for launching one transport pod. Pod launchers can launch as a group - but the launchers must be placed adjacent to each other. -->
-  <PodLauncher.description>TODO</PodLauncher.description>
+  <PodLauncher.description>Een brandstofdepot waarmee een enkele transportcapsule kan worden gelanceerd. Capsules kunnen in groepen worden gelanceerd, maar dan moeten de lanceerbases wel tegen elkaar aan worden gebouwd.</PodLauncher.description>
   
   <!-- EN: sarcophagus -->
-  <Sarcophagus.label>TODO</Sarcophagus.label>
+  <Sarcophagus.label>sarcofaag</Sarcophagus.label>
   <!-- EN: An ornamented burial receptacle for the honored dead. -->
-  <Sarcophagus.description>TODO</Sarcophagus.description>
+  <Sarcophagus.description>Een bewerkte grafkist bedoeld voor vereerde doden.</Sarcophagus.description>
   
   <!-- EN: grand stele -->
-  <SteleGrand.label>TODO</SteleGrand.label>
+  <SteleGrand.label>indruk-\nwekkende zuil</SteleGrand.label>
   <!-- EN: A towering slab with extensive engravings on the sides. Steles have been used since ancient times to memorialize individuals, battles, and other important events. -->
-  <SteleGrand.description>TODO</SteleGrand.description>
+  <SteleGrand.description>Een enorm stuk steen waarin gedetailleerd graveerwerk is aangebracht. Zuilen worden al eeuwenlang gebruikt om individuen, veldslagen en andere belangrijke gebeurtenissen vast te leggen.</SteleGrand.description>
   
   <!-- EN: large stele -->
-  <SteleLarge.label>TODO</SteleLarge.label>
+  <SteleLarge.label>grote zuil</SteleLarge.label>
   <!-- EN: A tall, thick slab with engravings on the sides. Steles have been used since ancient times to memorialize individuals, battles, and other important events. -->
-  <SteleLarge.description>TODO</SteleLarge.description>
+  <SteleLarge.description>Een groot stuk steen waarin graveerwerk is aangebracht. Zuilen worden al eeuwenlang gebruikt om individuen, veldslagen en andere belangrijke gebeurtenissen vast te leggen.</SteleLarge.description>
   
   <!-- EN: tool cabinet -->
-  <ToolCabinet.label>TODO</ToolCabinet.label>
+  <ToolCabinet.label>gereed-\nschapskast</ToolCabinet.label>
   <!-- EN: Increases work speed when placed near a workbench. One workbench use to up to two tool cabinets. -->
-  <ToolCabinet.description>TODO</ToolCabinet.description>
+  <ToolCabinet.description>Versnelt werk wanneer naast een werktafel geplaatst. Elke werkbank kan maximaal twee gereedschapskasten benutten.</ToolCabinet.description>
```

Comment:
Eerst werktafel, dan werkbank, consistent => ik heb niet per sé een voorkeur voor tafel of bank.

Reply:
Inderdaad, ik kies voor "werktafel", want dat wordt al elders gebruikt.